test/llilc_roslyn_stage.cmd

```diff
@@ -0,0 +1,25 @@
+REM Usage: dostage.cmd workspace buildsubdir stagedir
+REM 
+REM Will first delete (if present) and then create %workspace%/roslyn/%stagedir%
+REM directory and set it up as the directory
+REM from which Roslyn will run on CoreClr, with LLILC as JIT.
+
```

Comment:
can we start with setlocal and end with endlocal, or is the goal to have the environment variables permanently set?